asserts/asserts.go

```diff
@@ -63,8 +63,9 @@ var (
 
 // Assertion types without a definite authority set (on the wire and/or self-signed).
 var (
-	SerialProofType   = &AssertionType{"serial-proof", nil, assembleSerialProof, noAuthority}
-	SerialRequestType = &AssertionType{"serial-request", nil, assembleSerialRequest, noAuthority}
+	SerialProofType       = &AssertionType{"serial-proof", nil, assembleSerialProof, noAuthority}
+	SerialRequestType     = &AssertionType{"serial-request", nil, assembleSerialRequest, noAuthority}
+	AccountKeyRequestType = &AssertionType{"account-key-request", nil, assembleAccountKeyRequest, noAuthority}
```

Comment:
Agreed.  I think it ought to be reasonably straightforward to permit an authority-id later once we have clear requirements for that.